src/app/zap-templates/common/ClusterTestGeneration.js

```diff
@@ -26,7 +26,7 @@ const path              = require('path');
 // Import helpers from zap core
 const templateUtil = require(zapPath + 'dist/src-electron/generator/template-util.js')
 
-const { DelayCommands }                 = require('./simulated-clusters/TestDelayCommands.js');
+const { TestSuiteHelperCluster }        = require('./TestSuiteHelperCluster.js');
```

Comment:
"TestSuiteHelper" is too much of a generic name, it will end up as a holding bag of everything regardless of related or not. It says "helps tests" and will make code not easily readable in the long run since there  is no obvious mental model of 'helper' (evetyhing is a helper)